Home ourwork section

Genoux opened the pull request:
### Our work section 
Created the **Our work** section for the home page. 

---
**Added subsection**
- Header
- Case study (partial)
- Industries (partial linkcard)

---

Genoux committed:
Content structure of the 'our work' section (4)

Setting up the content structure in the _index.md for the 'our work' section in the home page.

Genoux committed:
Top content of 'our work' section

Genoux committed:
Basic case study partial structure

Built the case study partial so it can be use anywhere if you inject the content from the .md file

Genoux committed:
Industries subsection of 'our work' home section

Genoux committed:
Basic button

Genoux committed:
Building partial for link card

Link card is a block with a title and sometime a description that redirect to a URL on click

Genoux committed:
Cleaning and renaming

cleaning code format and renaming class

Genoux commented on assets/sass/pages/_home.scss:
```diff
@@ -1,55 +1,92 @@
+$sub-section-spacing: 100px 0px;
 #home-hero {
```
I'm not sure why this section got added in the PR. On my side spacing and indent are fine 🤔

afunnydev replied:
If you switch to 2 spaces indention it should be fine. Maybe you use tab?

afunnydev commented on assets/sass/main.scss:
```diff
@@ -7,6 +7,8 @@
 @import "base";
 @import "partials/header";
 @import "partials/footer";
+@import "partials/casestudy";
```
Keep a similar structure then the layouts, so `partials/site/casestudy`.

afunnydev commented on layouts/partials/site/linkcard.html:
```diff
@@ -0,0 +1,18 @@
+				{{- end -}}
+			</div>
+			<div class="link-arrow">
+				<svg width="20" height="20" viewBox="0 0 20 20" fill="none">
+					<path d="M4.55176 1H17.6935V13.7865" stroke="black" />
+					<path d="M17.6935 1L1 17.6935" stroke="black" />
+				</svg>
```
You can put this into `partials/icons/arrow-right.html` for reusage.

afunnydev commented on layouts/index.html:
```diff
@@ -41,4 +41,31 @@ <h2>{{- .subtitle -}}</h2>
+            <div class="row list">
+                {{- range .list -}}
+                <!-- Passing dict @name @info @url content to the link card partial -->
+                {{- partial "site/linkcard" (dict "name" .name "info" "" "url" "#") -}}
```
Add `"context" .` as the first item in your dict. It's a good habit to take, and it makes sure you pass the general Hugo `.` to the partial (in case you need something like `.Site.BaseURL`).

afunnydev commented:
I just realized that I messed up the `layouts/partials` folder on setup. Everything currently (pre-PR) in `layouts/partials/site` should actually be in `layouts/partials`. We will use `layouts/partials/site` for components (like you did, that was perfect).

Genoux commented:
> 
> 
> I just realized that I messed up the `layouts/partials` folder on setup. Everything currently (pre-PR) in `layouts/partials/site` should actually be in `layouts/partials`. We will use `layouts/partials/site` for components (like you did, that was perfect).

Do you want me to restructure those folders on this PR?

afunnydev commented:
Yes, please!

Genoux committed:
Adding context to the linkcard dictionary

Genoux committed:
Created an arrow icon as partial for reusability

Genoux committed:
Changed the partials folder structure

Core partials (e.g. header, script, etc) will be in the partials root folder. All other components will be in partials/site/ folder

Genoux committed:
Cleaning font styling and size

Cleaning unnecessary font styling and change font size according to Figma design. (We should probably think about integrating variable for different P scenario, like p1, p2, p3, etc)

afunnydev commented:
@Genoux Are we all set for a merge?

Genoux commented:
> 
> 
> @Genoux Are we all set for a final review?

Yep! If it's all good with you!

afunnydev commented:
Great work! There are some small optimizations to do. I will merge (for simplicity), but I'll commit some changes and share to commit with you so that you can have a look!
